lib/compiler/aro/aro/Preprocessor.zig

```diff
@@ -389,7 +389,8 @@ fn preprocessExtra(pp: *Preprocessor, source: Source) MacroError!TokenWithExpans
     try pp.ensureTotalTokenCapacity(pp.tokens.len + estimated_token_count);
 
     var if_level: u8 = 0;
-    var if_kind: [64]u8 = .{0} ** 64;
+    const if_kind_buf_size = comptime mem.packedArrayByteLen(u2, 256) catch unreachable;
+    var if_kind: [if_kind_buf_size]u8 = .{0} ** if_kind_buf_size;
```

Comment:
I'm aware that this code is out-of-sync with upstream Aro. Will submit an upstream PR if this is merged.

Reply:
Would `var if_kind: [if_kind_buf_size]u8 = @splat(0);` be better here?